test/gtest/uct/test_p2p_rma.cc

```diff
@@ -151,4 +151,44 @@ UCS_TEST_SKIP_COND_P(uct_p2p_rma_test, madvise,
     flush();
 }
 
+UCS_TEST_SKIP_COND_P(uct_p2p_rma_test, fork,
+                     !check_caps(UCT_IFACE_FLAG_GET_ZCOPY),
+                     "RCACHE_CHECK_PFN=1")
+{
+    char *page = (char *)valloc(300);
```

Comment:
why use valloc?
if we need page-aligned mem, can let mapped_buffer use entity::mem_alloc_host and set "UCX_ALLOC=mmap", this will not require changes in mapped_buffer